src/createTwitchVideoCompiler/youtubeUploader.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+
+import fs from 'fs';
+import readline from 'readline';
+import { google } from 'googleapis';
+
+export default async function youtubeUploader(
+  clientFilePath: string,
+  properties: {
+    language: string,
+    description: string,
+    tags: string,
+    title: string,
+    privacy: string,
+    videoPath: string,
+  },
+): Promise<void> {
+  return new Promise((resolve, reject) => {
```

Comment:
Do not put the whole thing in the [executor function](https://blog.domenic.me/the-revealing-constructor-pattern/), only what's necessary.